versions/3.0.md

```diff
@@ -311,7 +311,7 @@ url: http://www.apache.org/licenses/LICENSE-2.0.html
 
 #### <a name="serverObject"></a>Server Object
 
-An object representing a Server.
+An object representing a list of one or more servers.
```

Comment:
This is not true. It represents a single server. The example for the object is wrong and needs to be updated.